<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SubirArchivoButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subir Archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubirCarpetaButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Subir Carpeta</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="ArchivosTree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Tus Archivos</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ActualizarTodoButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>283</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar Todo</string>
   </property>
  </widget>
  <widget class="QListWidget" name="AmigosList">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>360</y>
     <width>181</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TusAmigosLabel">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>330</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tus Amigos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ConversarButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>650</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Conversar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ChatTextField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Escribe aqui...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnviarButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>630</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConversandoConLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Conversando con: Nadie</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>681</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="ChatList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
  </widget>
  <widget class="QLabel" name="UsuarioConectadoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Usuario conectado: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BajarArchivoButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bajar Archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BajarCarpetaButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bajar Carpeta</string>
   </property>
  </widget>
  <widget class="QLabel" name="AgregarAmigoLabel">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>330</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agregar Amigo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="AgregarAmigoLineEdit">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>360</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AgregarAmigoButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnviarArchivoButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>650</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar Archivo</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>20</width>
     <height>681</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="IconoLabel">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>440</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>ICONO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CargandoLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NotificacionLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>NOTIFICACION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>21</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="NotificacionButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="NotificacionTextLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&quot;javier&quot; desea enviarle el archivo &quot;foto.png&quot;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarArchivoButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RenombrarArchivoButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Renombrar Archivo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NuevoNombreArchivo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>nuevo_nombre.abc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VerHistorialButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Historial</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="HistorialList">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
  </widget>
  <widget class="QLabel" name="HistorialLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Historial de Modificaciones</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarCarpetaButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Carpeta</string>
   </property>
  </widget>
  <zorder>SubirArchivoButton</zorder>
  <zorder>SubirCarpetaButton</zorder>
  <zorder>ArchivosTree</zorder>
  <zorder>ActualizarTodoButton</zorder>
  <zorder>AmigosList</zorder>
  <zorder>TusAmigosLabel</zorder>
  <zorder>ConversarButton</zorder>
  <zorder>ChatTextField</zorder>
  <zorder>EnviarButton</zorder>
  <zorder>ConversandoConLabel</zorder>
  <zorder>line</zorder>
  <zorder>ChatList</zorder>
  <zorder>UsuarioConectadoLabel</zorder>
  <zorder>BajarArchivoButton</zorder>
  <zorder>BajarCarpetaButton</zorder>
  <zorder>AgregarAmigoLabel</zorder>
  <zorder>AgregarAmigoLineEdit</zorder>
  <zorder>AgregarAmigoButton</zorder>
  <zorder>EnviarArchivoButton</zorder>
  <zorder>line_2</zorder>
  <zorder>IconoLabel</zorder>
  <zorder>CargandoLabel</zorder>
  <zorder>EliminarArchivoButton</zorder>
  <zorder>RenombrarArchivoButton</zorder>
  <zorder>NuevoNombreArchivo</zorder>
  <zorder>VerHistorialButton</zorder>
  <zorder>line_3</zorder>
  <zorder>HistorialList</zorder>
  <zorder>HistorialLabel</zorder>
  <zorder>NotificacionLabel</zorder>
  <zorder>NotificacionButton</zorder>
  <zorder>NotificacionTextLabel</zorder>
  <zorder>EliminarCarpetaButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
